File: 10 - Python/Exercícios/04 - Vetores/Exercício 07.py
#Exercício - Abaixo da Média

soma = 0

N = int(input("Quantos elementos vai ter o vetor? "))

vet = [0] * N

for i in range(N):

    vet[i] = float(input("Digite um número: "))
    soma = soma + vet[i]

media = soma / N

print("")
print("Média do vetor =", round(media, 3))
print("Elementos abaixo da média:")

for i in range(N):

    if vet[i] < media:
        print(vet[i])

File: 07 - C++/Exercícios/04 - Vetores/Exercício 07.cpp
//Exercício - Abaixo da Média

#include <iostream>
#include <iomanip>

using namespace std;

int main(){

    int i, N;
    double soma, media;

    soma = 0;

    cout << "Quantos elementos vai ter o vetor? ";
    cin >> N;

    double vet[N];

    for(i = 0; i < N; i++){

        cout << "Digite um numero: ";
        cin >> vet[i];

        soma = soma + vet[i];
    }

    media = soma / N;
    
    cout << fixed << setprecision(3);
    cout << endl << "Media do vetor = " << media << endl;

    cout << fixed << setprecision(1);
    cout << "Elementos abaixo da media:" << endl;
    for(i = 0; i < N; i++){
        if(vet[i] < media){
            cout << vet[i] << endl;
        }
    }

    return 0;

}